release/staging-resources-cdk/bin/staging-resources-cdk.ts

```diff
@@ -18,6 +19,8 @@ const gitHubAccessStack = new GitHubAccessStack(app, 'GitHubAccessStack', {
   stackName: 'GitHubAccess'
 });
 
+new ArtifactsBucketStack(app, 'ArtifactsBucketStack');
```

Comment:
Just curious, why did you not specify the `stackName` like you did for all other stacks?

Reply:
The other stacks have some global impact and it would be invalid to create multiple of them. This stack just creates an S3 bucket. I thought that CDK would create a CloudFormation stack name of `ArtifactsBucketStack{someUniqueCharacters}` in this case. So you could potentially create multiple buckets. But, the actual CloudFormation stack name is `ArtifactsBucketStack`. I'll add the `stackName` here for consistency.